request-manager/resources/request_manager_schema.sql

```diff
@@ -24,18 +24,20 @@ CREATE TABLE IF NOT EXISTS `raw_requests` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
 CREATE TABLE IF NOT EXISTS `job_log` (
-  `request_id`  BINARY(20) NOT NULL,
-  `job_id`      BINARY(4) NOT NULL,
-  `name`        VARBINARY(100) NOT NULL,
-  `try`         SMALLINT NOT NULL DEFAULT 0,
-  `type`        VARBINARY(75) NOT NULL,
-  `state`       TINYINT UNSIGNED NOT NULL DEFAULT 0,
-  `started_at`  BIGINT UNSIGNED NOT NULL DEFAULT 0, -- Unix time (nanoseconds)
-  `finished_at` BIGINT UNSIGNED NOT NULL DEFAULT 0, -- Unix time (nanoseconds)
-  `error`       TEXT NULL DEFAULT NULL,
-  `exit`        TINYINT UNSIGNED NULL DEFAULT NULL,
-  `stdout`      LONGBLOB NULL DEFAULT NULL,
-  `stderr`      LONGBLOB NULL DEFAULT NULL,
+  `request_id`   BINARY(20) NOT NULL,
+  `job_id`       BINARY(4) NOT NULL,
+  `name`         VARBINARY(100) NOT NULL,
+  `try`          SMALLINT NOT NULL DEFAULT 0,
+  `sequence_id`  BINARY(4) NOT NULL, -- TODO(alyssa) figure out default value
```

Comment:
it's possible this may be empty, for example the first job in the graph does not have a parent sequence id. idk what the default should be. can i just put `DEFAULT 0`? @michaelfinch

Reply:
I thought about this some more and i think this exposes a design flaw in my solution. all jobs are part of a sequence by definition, so there should be no case in which a job does not have a sequence id. to solve this, i will have `sequence_id` and `parent_sequence_id`. this just becomes a little tricky when i record the job in the job log. if a job is part of a sub sequence in its parent sequence, but we retry the parent sequence, we want to log that this job is being retried as part of the parent sequence. will look at code and see how to make it happen.

Reply:
Does `sequence_id` tell us anything meaningful, i.e. to a human? Job IDs are mostly a necessity for establishing a short primary key to identify rows in this table, but to a human, what about a sequence do we want to know wrt a job? How would I ever look up or know what `sequence_id=x9r4` is?

Reply:
a job can be part of many sequences. it may be retried in the immediate sequence that it is part of or it may be retried as part of one of its parent's sequences. so, a job sequence retry count meaningless unless we know what sequence initiated the retry. This will help us differentiate a single job that was part of distinct sequence retry runs.